server/genRecs.js

```diff
@@ -27,9 +28,11 @@ const getDists = (userData, cb) => {
       }
       distances.push([i, eucDist(userData.profile, res)]);
       if (i === 300000) {
-        cb(distances.sort((a, b) => (a[1] - b[1]))
-          .slice(0, 20)
-          .map(movie => (movie[0])));
+        timsort.sort(distances, (a, b) => (a[1] - b[1]))
```

Comment:
How efficient is timsort compared to javascript native sort method?

Reply:
Wikipedia has a great breakdown of common sorting algorithms and time complexities: https://en.wikipedia.org/wiki/Sorting_algorithm
(Javascript's native sort method uses Quicksort)

The npm module I used for timsort also has a breakdown of how the execution of Timsort compares to native sort: https://www.npmjs.com/package/timsort